<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionsForPeriod</class>
 <widget class="QDialog" name="ActionsForPeriod">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QLabel" name="lblPeriodDuration">
       <property name="text">
        <string>Длительность мероприятия (дней)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lblPeriodInterval">
       <property name="text">
        <string>Интервал между мероприятиями (дней)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="etdAmount">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="lblPeriodBegDate">
       <property name="text">
        <string>Начало периода</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="CDateEdit" name="edtPeriodEndDate"/>
     </item>
     <item row="1" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="lblPeriodEndDate">
       <property name="text">
        <string>Окончание периода</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="CDateEdit" name="edtPeriodBegDate"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblAmount">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Количество</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="edtInterval">
       <property name="maximum">
        <number>9999999</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="edtDuration">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QCheckBox" name="chkFillType">
       <property name="text">
        <string>Заполнение по количеству</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QCheckBox" name="chkWeekend">
       <property name="text">
        <string>Учитывать выходные</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtPeriodBegDate</tabstop>
  <tabstop>edtPeriodEndDate</tabstop>
  <tabstop>etdAmount</tabstop>
  <tabstop>chkFillType</tabstop>
  <tabstop>edtDuration</tabstop>
  <tabstop>edtInterval</tabstop>
  <tabstop>chkWeekend</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ActionsForPeriod</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ActionsForPeriod</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkFillType</sender>
   <signal>toggled(bool)</signal>
   <receiver>etdAmount</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkFillType</sender>
   <signal>toggled(bool)</signal>
   <receiver>lblAmount</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkFillType</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtPeriodEndDate</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkFillType</sender>
   <signal>toggled(bool)</signal>
   <receiver>lblPeriodEndDate</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
